src/Form/Theme.php

```diff
@@ -19,6 +19,17 @@
  * @since 2.7.0
  */
 abstract class Theme {
+	/**
+	 * view vs class array
+	 *
+	 * @since 2.7.0
+	 * @var array
+	 */
+	public $view = [
```

Comment:
Please pluralize this: `views`

Reply:
fixed: https://github.com/impress-org/givewp/compare/667bf05cc9e7...8584bd4f57e9